Added the READme file (#3)
aicoe-aiops/HTM-applications: README.md

# HTM-Applications

> The Hierarchical Temporal Memory (HTM) algorithm is based on the well understood principles and core building blocks of the [Thousand Brains Theory](https://link.springer.com/content/pdf/10.1007/s42452-021-04715-0.pdf), which is the core-model based, sensory-motor framework of intelligence putting together the neuroscience research developed at [Numenta](https://numenta.com/) founded by Jeff hawkings. It aims to reflect the functioning of the human cortex. It focusses on three main properties: _sequence learning_, _continual learning_, and _sparse distributed representations_.

Reviewer: ```suggestion
The Hierarchical Temporal Memory (HTM) algorithm is based on the well understood principles and core building blocks of the [Thousand Brains Theory](https://link.springer.com/content/pdf/10.1007/s42452-021-04715-0.pdf), which is the core-model based, sensory-motor framework of intelligence putting together the neuroscience research developed at [Numenta](https://numenta.com/) founded by Jeff Hawkins. It aims to reflect the functioning of the human cortex. It focuses on three main properties: _sequence learning_, _continual learning_, and _sparse distributed representations_.
```

> The project aims to explore the applications of HTM primarily focussing on [Anomaly Detection](https://numenta.com/assets/pdf/whitepapers/Numenta%20White%20Paper%20-%20Science%20of%20Anomaly%20Detection.pdf) technique.

Reviewer: ```suggestion
This project aims to explore the applications of HTM, specifically its [Anomaly Detection](https://numenta.com/assets/pdf/whitepapers/Numenta%20White%20Paper%20-%20Science%20of%20Anomaly%20Detection.pdf) technique, in the AIOps domain.
```
